vmdk_plugin/utils/refcount/refcnt.go

```diff
@@ -181,7 +203,7 @@ func (r RefCountsMap) Init(d drivers.VolumeDriver, mountDir string, name string)
 	err = r.discoverAndSync(c, d)
```

Comment:
(1) is there  indeed a deadlock generated after docker API is initialized() ? 
If we did not check (the code or a test) , a small test is needed (e.g. recount or getMountedVolumesFromDocker() inside of a Create() would do it). @pshahzeb 

(2) In case the deadlock is indeed generated: for 2 maps suggestions (which I like because we ended up doing the same for DS cache in the server side :-)). how are we planning to track changes to the actual bitmap done while AFTER the "new" one slot is filled but BEFORE the maps were swapped ? 

(3) Since we can wait a couple of minutes if we deadlock on start, why not do the same if we deadlock later on ? Similar to ethernet conflict resolution :-) - "back off and retry in a random amount of time (within min/max)" ?